Audit logging is enabled.
The current configuration properties are:
AuditLogProperty1=AuditLogValue1
AuditLogProperty2=AuditLogValue2
AuditLogProperty3=AuditLogValue3

The current registered provider modules are:
AuditLogProviderModule0=Unknown
AuditLogProviderModule1=Other
AuditLogProviderModule2=OK
AuditLogProviderModule3=Degraded
AuditLogProviderModule4=Stressed
AuditLogProviderModule5=Predictive Failure
AuditLogProviderModule6=Error
AuditLogProviderModule7=Non-Recoverable Error
AuditLogProviderModule8=Starting
AuditLogProviderModule9=Stopping
AuditLogProviderModule10=Stopped
AuditLogProviderModule11=In Service
AuditLogProviderModule12=No Contact
AuditLogProviderModule13=Lost Communication

The planned value of property "logdir" is modified from value "./logs" to value "/tmp" by user "guest".
The current value of property "logdir" is modified from value "./logs" to value "/tmp" by user "guest".
A CIM CreateClass operation on class "class1" in namespace "root/test" by user "user1" connected from system "127.0.0.1" resulted in status "CIM_ERR_FAILED: A general error occurred that is not covered by a more specific error code".
A CIM ModifyClass operation on class "CIM_MyClass" in namespace "test" by user "" connected from system "localhost" resulted in status "CIM_ERR_ACCESS_DENIED: Access to a CIM resource was not available to the client".
A CIM DeleteClass operation on class "ByeByeClass" in namespace "a/b/c/d" by user "guest" connected from system "1.1.1.1" resulted in status "CIM_ERR_SUCCESS: successful".
A CIM SetQualifier operation on qualifier "Test" in namespace "test2" by user "rootabc" connected from system "255.0.0.255" resulted in status "CIM_ERR_NOT_SUPPORTED: The requested operation is not supported".
A CIM DeleteQualifier operation on qualifier "Bogus" in namespace "test1/test2" by user "r" connected from system "4.3.2.1" resulted in status "CIM_ERR_NOT_FOUND: The requested object could not be found".
A CIM CreateInstance operation on instance "class1" in namespace "root/test" by user "user1" connected from system "127.0.0.9" resulted in status "CIM_ERR_FAILED: A general error occurred that is not covered by a more specific error code".  
A CIM CreateInstance operation on instance "class1.id="v1"" in namespace "root/test" by user "user1" connected from system "127.0.0.9" resulted in status "CIM_ERR_SUCCESS: successful".  The provider for this operation is "Provider1" in module "Module1".
A CIM ModifyInstance operation on instance "CIM_MyClass.id=1" in namespace "test" by user "" connected from system "localhost" resulted in status "CIM_ERR_INVALID_CLASS: The specified class does not exist".  
A CIM ModifyInstance operation on instance "CIM_MyClass.id=1" in namespace "test" by user "" connected from system "localhost" resulted in status "CIM_ERR_INVALID_CLASS: The specified class does not exist".  The provider for this operation is "TheProvider" in module "TheModule".
A CIM DeleteInstance operation on instance "THE_Class.a="A",z=10" in namespace "a/b/c" by user "guest" connected from system "127.0.0.1" resulted in status "CIM_ERR_NOT_FOUND: The requested object could not be found".  
A CIM DeleteInstance operation on instance "THE_Class.a="A",z=10" in namespace "a/b/c" by user "guest" connected from system "127.0.0.1" resulted in status "CIM_ERR_NOT_FOUND: The requested object could not be found".  The provider for this operation is "AHappyProvider" in module "AHappyModule".
A CIM SetProperty operation on instance "Z.x=0" in namespace "a/b/c/d/e/f/g" by user "me" connected from system "1.0.0.0" resulted in status "CIM_ERR_INVALID_CLASS: The specified class does not exist".  
A CIM SetProperty operation on instance "Z.x=0" in namespace "a/b/c/d/e/f/g" by user "me" connected from system "1.0.0.0" resulted in status "CIM_ERR_INVALID_CLASS: The specified class does not exist".  The provider for this operation is "InstProv" in module "InstModule".
A CIM InvokeMethod operation on method "MyMethod" of object "Test" in namespace "t" by user "guest" connected from system "1.1.1.1" resulted in status "CIM_ERR_FAILED: A general error occurred that is not covered by a more specific error code".
A CIM InvokeMethod operation on method "MyMethod" of object "Test.a=1,b=2,c=3" in namespace "t" by user "guest" connected from system "1.1.1.1" resulted in status "CIM_ERR_FAILED: A general error occurred that is not covered by a more specific error code".  The provider for this operation is "p" in module "m".
Audit logging is disabled.
